bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -252,6 +263,11 @@ public boolean isCompletionMarker(Result[] result) {
     public long getRowCount(Result[] result) {
       return result == null ? 0 : result.length;
     }
+
+    @Override
+    public ByteString getKey(Result[] result) {
+      return isCompletionMarker(result) ? null : ByteString.copyFrom(result[result.length - 1].getRow());
```

Comment:
`isCompletionMarker()` appears to check for `result == null` but I don't see the check for `result.length == 0` here or there.